index.html

```diff
@@ -37,6 +34,223 @@ <h4 class="profession">I am a Software Developer</h4>
       <li class="list-social"><a class="logos"  href="#"><img src="img/twitter.svg" alt="Twitter Logo"></a></li>
       <li class="list-social"><a class="logos"  href="#"><img src="img/medium.png" alt="Medium Logo"></a></li>
     </ul>         
-  </section>  
+  </section> 
+  
+  <section class="works">
+    <header class="container-header-works">
+      <h3 class="header-works">My Recent Works</h3>
+      <hr class="division division-line1"></hr>
```

Comment:
Great job so far 👍🏼 , I want just to mention that this divider is not well aligned kindly refer to the images below and fix the  issue 
![image (31)](https://github.com/eamador1/portfolio-project/assets/59707859/3de5f64d-a361-40cb-9a43-02a6fcfc591c)
![image (32)](https://github.com/eamador1/portfolio-project/assets/59707859/e2c4d63f-56e8-4269-9826-b8d72dbf0ad6)